shared/featureconfig/config.go

```diff
@@ -126,14 +124,6 @@ func ConfigureBeaconChain(ctx *cli.Context) {
 		log.Warn("Enabling experimental kafka streaming.")
 		cfg.KafkaBootstrapServers = ctx.GlobalString(kafkaBootstrapServersFlag.Name)
 	}
-	if ctx.GlobalBool(enableCommitteeCacheFlag.Name) {
```

Comment:
shouldn't we support it still if it is in deprecated?

Reply:
Not sure if I understand the question. These are not used